src/Units/Weapons/WeaponModel.java

```diff
@@ -54,29 +58,35 @@ public int getIndex()
-/* ROCKETS          */{  95,   90,    90,   80,      55,      50,  80,        80,      85,       85,        90,       0,      0,        0,        0,         55,      85,     60,  85,       0 }, // ROCKETS
-/* ANTI_AIRMGUN     */{ 105,  105,    60,   25,      10,       5,  50,        50,      55,       45,        55,      65,     75,      120,      120,          0,       0,      0,   0,       0 }, // ANTI_AIRMGUN
-/* MOBILESAM        */{   0,    0,     0,    0,       0,       0,   0,         0,       0,        0,         0,     100,    100,      120,      120,          0,       0,      0,   0,       0 }, // MOBILESAM
-/* FIGHTERMISSILES  */{   0,    0,     0,    0,       0,       0,   0,         0,       0,        0,         0,      55,    100,      100,      100,          0,       0,      0,   0,       0 }, // FIGHTERMISSILES
-/* BOMBERBOMBS      */{ 110,  110,   105,  105,      95,      90, 105,       105,     105,       95,       105,       0,      0,        0,        0,         75,      85,     95,  95,       0 }, // BOMBERBOMBS
-/* B_COPTERROCKETS  */{   0,    0,    55,   55,      25,      20,  60,        65,      65,       25,        65,       0,      0,        0,        0,         25,      55,     25,  25,       0 }, // B_COPTERROCKETS
-/* B_COPTERMGUN     */{  75,   75,    30,    6,       1,       1,  20,        25,      35,        6,        35,       0,      0,       65,       95,          0,       0,      0,   0,       0 }, // B_COPTERMGUN
-/* BATTLESHIPCANNON */{  95,   90,    90,   80,      55,      50,  80,        80,      85,       85,        90,       0,      0,        0,        0,         50,      95,     95,  95,       0 }, // BATTLESHIPCANNON
-/* CRUISERTORPEDOES */{   0,    0,     0,    0,       0,       0,   0,         0,       0,        0,         0,       0,      0,        0,        0,          0,       0,      0,  90,      90 }, // CRUISERTORPEDOES
-/* CRUISERMGUN      */{   0,    0,     0,    0,       0,       0,   0,         0,       0,        0,         0,      55,     65,      115,      115,          0,       0,      0,   0,       0 }, // CRUISERMGUN
-/* SUBTORPEDOES     */{   0,    0,     0,    0,       0,       0,   0,         0,       0,        0,         0,       0,      0,        0,        0,         55,      25,     95,  55,      55 }  // SUBTORPEDOES
+// defenders:      INFANTRY, MECH, RECON, TANK, MD_TANK, NEOTANK,           APC, ARTILLERY, ROCKETS,             ANTI_AIR, MOBILESAM, FIGHTER, BOMBER,                        B_COPTER, T_COPTER,                        BATTLESHIP, CRUISER, LANDER, SUB, SUB_SUB
+// defenders:      INFANTRY, MECH, RECON, TANK, MD_TANK, NEOTANK, MEGATANK, APC, ARTILLERY, ROCKETS, PIPERUNNER, ANTI_AIR, MOBILESAM, FIGHTER, BOMBER, STEALTH, STEALTH_HIDE, B_COPTER, T_COPTER, BBOMB, CARRIER, BBOAT, BATTLESHIP, CRUISER, LANDER, SUB, SUB_SUB
```

Comment:
Did you mean to (mostly) double up the "defenders" line here?

Reply:
...That was a guide for adding the new rows. Which I then forgot to remove.
That knowledge might help you review the chart changes, if you care to?

Planning to delete in the `mod` branch.